### PR TITLE
Initial project

### PR DESCRIPTION
**Pull request by innomaxx**

### CODE REVIEW

File: src/main/java/com/max/grpc/orders/client/CafeClient.java [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+import java.util.concurrent.TimeUnit;
+
+public class CafeClient {
+    private Logger logger = Logger.getLogger(CafeClient.class);
+    private ManagedChannel managedChannel;
+    private MenuServiceBlockingStub menuStub;
+    private OrderServiceBlockingStub orderStub;
+
+    public CafeClient(String host, int port) {
+        var channel = ManagedChannelBuilder.forAddress(host, port).usePlaintext().build();
```
Comment (GalynaY) [inline src/main/java/com/max/grpc/orders/client/CafeClient.java]: Why?????

File: src/main/java/com/max/grpc/orders/client/CafeClient.java [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+import io.grpc.StatusRuntimeException;
+
+import org.apache.log4j.Logger;
+
+import java.util.List;
+import java.util.concurrent.TimeUnit;
+
+public class CafeClient {
+    private Logger logger = Logger.getLogger(CafeClient.class);
+    private ManagedChannel managedChannel;
```
Comment (GalynaY) [inline src/main/java/com/max/grpc/orders/client/CafeClient.java]: Do final

File: src/main/java/com/max/grpc/orders/client/CafeClient.java (hunk 1 above)
Comment (innomaxx) [reply src/main/java/com/max/grpc/orders/client/CafeClient.java]: Removed

File: src/main/java/com/max/grpc/orders/client/CafeClient.java (hunk 2 above)
Comment (innomaxx) [reply src/main/java/com/max/grpc/orders/client/CafeClient.java]: Done for all fields

**Pull request merged**